app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    cp, file, file_copy = command.split(" ")
```

Comment:
cp is not descriptive name for variable

Reply:
Also no need to pass " " in split method it use spaces by default

Reply:
Fixed